### ISSUE TITLE
Can shell script use python3 only?

### ISSUE THREAD

**Issue by pamelafox** [Arize-ai/phoenix-on-azure#11]
I'm on a Mac, and my `python` command doesn't actually work, due to a poorly setup alias. The `python3` command always works, however. We only use `python3` in our .sh scripts in other azd templates, so I wonder if that'd be possible? Or are there Windows environments running the shell scripts that don't have python3?

**Comment by abalam666** [comment]
Hello @pamelafox !
Thank you for your contribution, you read our minds!
We did [this commit](https://github.com/Arize-ai/phoenix-on-azure/commit/2000f9b863af7439bcbe0c6dc73a6cbc1cb5afa3) two days ago !
Is it what you're asking for ?

**Comment by pamelafox** [comment]
That's the version I was using, but it errors on my machine, due to my python alias being messed up. I should probably fix my alias, but I've never bothered since I always type python3

My experience:

```
phoenix-on-azure % ./scripts/init.sh   
pyenv: python: command not found
e
The `python' command exists in these Python versions:
  3.9.14

Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
```

However, I realized I needed to setup a venv first anyway, since you require a pip install, and python command works inside a venv, so I was able to get it working after all.

I'm just curious if there's some environment which calls shell scripts which doesn't have the python3 alias, as our shell scripts in other repos only use python3, and I don't think I've gotten bug reports baout them.  (I know that Windows doesn't have it, but I assume ps1 is running on Windows)
